src/api/draftSpecs.ts

```diff
@@ -42,7 +42,8 @@ export const createDraftSpec = (
     catalogName: string,
     draftSpec: any,
     specType?: Entity | null,
-    lastPubId?: string | null
+    lastPubId?: string | null,
+    noResponse?: boolean
```

Comment:
A simple way to make sure extra data is not returned. This matches how the newer API for Supabase works. However, a lot of code relies on it so making it a flag for now.